sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -1750,6 +1763,12 @@ public partial class SearchIndexer
         public string SkillsetName { get { throw null; } set { } }
         public string TargetIndexName { get { throw null; } set { } }
     }
+    public partial class SearchIndexerCache
+    {
+        public SearchIndexerCache() { }
+        public bool? EnableReprocessing { get { throw null; } set { } }
+        public string StorageConnectionString { get { throw null; } set { } }
```

Comment:
Should this be publicly exposed as a property?  Or should we just have a `SetStorageConnectionString` method (i.e., kind of make it a write-only property).

Reply:
What benefit will that give us?
Will we do some validation/other kind of processing in the setter method?

Reply:
Similar to the rationale we use for not exposing the password on our credential types - it's secret and nobody should be reading it here.  We can't prevent a malicious attacker from doing so (i.e., [even SecureString isn't all that secure](https://github.com/dotnet/platform-compat/blob/master/docs/DE0001.md)), but we can help people avoid doing dangerous things by only making it possible to set the value.